dbt_platform_helper/commands/config.py

```diff
@@ -156,21 +158,27 @@ def deployment():
     exit(0 if compatible else 1)
 
 
-def tool_versions():
+def _check_tool_versions_and_return_platform_helper_versions():
```

Comment:
This method is doing a lot.

I think it might be worth refactoring this into two functions...

`get_platform_helper_versions() -> PlatformHelperVersions`

...and...

`check_platform_helper_versions(versions: PlatformHelperVersions) -> None`

...methods to get closer to a single responsibility vibe.